modules/oe_translation_corporate_workflow/modules/oe_translation_active_revision/tests/src/FunctionalJavascript/ActiveRevisionTest.php

```diff
@@ -0,0 +1,573 @@
+      $node = $this->moderateNode($node, 'published');
+
+      $node->addTranslation('fr', ['title' => 'My FR node']);
+      $node->save();
+
+      // Keep track of the version 1 revision ID.
+      $version_one_revision_id = $node->getRevisionId();
+
+      $this->drupalGet($node->toUrl());
+      $this->assertSession()->pageTextContains('My node');
+      $this->assertSession()->pageTextContains('Non translatable value');
+
+      $this->drupalGet('/fr/node/' . $node->id());
+      $this->assertSession()->pageTextContains('My FR node');
+      $this->assertSession()->pageTextContains('Non translatable value');
+
+      // Start a new draft.
+      $node->set('title', 'My updated node');
+      $node->set('field_non_translatable_field', 'Non translatable updated value');
+      $node->set('moderation_state', 'draft');
+      $node->setNewRevision();
+      $node->save();
+
+      $this->drupalGet($node->toUrl('latest-version'));
+      $this->assertSession()->fieldExists('The new version needs NEW translations');
+      $this->assertSession()->fieldExists('The new version does NOT need NEW translations (there have been only changes to non-translatable fields)');
+      $this->assertFalse($this->getSession()->getPage()->findField('The new version needs NEW translations')->isVisible());
+      $this->getSession()->getPage()->selectFieldOption('Change to', $state);
+      $this->assertTrue($this->getSession()->getPage()->findField('The new version needs NEW translations')->isVisible());
+      $this->assertTrue($this->getSession()->getPage()->findField('The new version needs NEW translations')->isChecked());
+      $this->assertTrue($this->getSession()->getPage()->findField('The new version does NOT need NEW translations (there have been only changes to non-translatable fields)')->isVisible());
+      $this->assertFalse($this->getSession()->getPage()->findField('The new version does NOT need NEW translations (there have been only changes to non-translatable fields)')->isChecked());
+      $this->assertTrue($this->getSession()->getPage()->findField('Keep current translations until new ones are synchronised.')->isVisible());
+      $this->assertTrue($this->getSession()->getPage()->findField('Keep current translations until new ones are synchronised.')->isChecked());
+      $this->assertTrue($this->getSession()->getPage()->findField('Delete current translations until new ones are synchronized')->isVisible());
+      $this->assertFalse($this->getSession()->getPage()->findField('Delete current translations until new ones are synchronized')->isChecked());
+
+      // Assert the state changes the other options.
+      $this->getSession()->getPage()->findField('The new version does NOT need NEW translations (there have been only changes to non-translatable fields)')->click();
+      $this->assertFalse($this->getSession()->getPage()->findField('Keep current translations until new ones are synchronised.')->isVisible());
```

Comment:
We may again assert that `Delete current translations until new ones are synchronized` is also not visible.
```suggestion
      $this->assertFalse($this->getSession()->getPage()->findField('Keep current translations until new ones are synchronised.')->isVisible());
      $this->assertFalse($this->getSession()->getPage()->findField('Delete current translations until new ones are synchronized')->isVisible());
```